src/structures/ThreadMember.js

```diff
@@ -8,10 +8,6 @@ const ThreadMemberFlags = require('../util/ThreadMemberFlags');
  * @extends {Base}
  */
 class ThreadMember extends Base {
-  /**
-   * @param {ThreadChannel} thread The thread that this member is associated with
-   * @param {APIThreadMember} data The data for the thread member
```

Comment:
This was the last instance of `APIThreadMember`. Should we remove this too?

https://github.com/discordjs/discord.js/blob/395186da7241b1ea704e4d5e0c6df6fd4655eb5c/src/structures/ThreadMember.js#L94-L97